_includes/title.html

```diff
@@ -19,10 +19,14 @@
     <h1>
         {{ page.title | xml_escape }}
         <small>{{ page.subtitle }}</small>
+        
         {% if edit_path %}
             <a class="btn btn-primary pull-right" href="{{ edit_path }}">Edit on Github</a>
         {% elsif show_news_link %}
             <a class="btn btn-primary pull-right" href="/news/atom.xml">Subscribe via Atom</a>
         {% endif %}
     </h1>
+    {% if page.author %}
```

Comment:
I see the issue. Currently, I expect that this addition will cause this to show up on project pages too (where we already use an `author`property), which would be duplicating information. For now, can you just rename this to `tutorial_author`, `docs_author` or similar so we avoid the collision?

As for the box, have you taken a look at the project pages? We already have that info box functional if we'd like it here. The problem is that it takes up a fair amount of horizontal space. So I'd say leave it as-is for now, and when I get the chance I can try to modify it to make it work here without taking up too much space.